src/components/button/button.ts

```diff
@@ -139,13 +160,15 @@ export class Button extends LitElement {
       [`kd-btn--${baseTypeClass}`]: !this.destructive,
       'kd-btn--large': this.size === BUTTON_SIZES.LARGE,
       'kd-btn--small': this.size === BUTTON_SIZES.SMALL,
-      'kd-btn--medium': this.size === BUTTON_SIZES.MEDIUM,
+      'kd-btn--medium': this.reSizeBtn || this.size === BUTTON_SIZES.MEDIUM,
       [`kd-btn--icon-${this.iconPosition}`]:
         !!this.iconPosition && !this.iconOnly,
       [`kd-btn--icon-center`]: this._iconEls?.length && this.iconOnly,
       'icon-only': this._iconEls?.length && this.iconOnly,
+      'btn-float': this.isFloating,
+      'btn-hidden': this.showOnScroll && !this.showButton,
+      'btn-visible': !this.showOnScroll || this.showButton,
```

Comment:
`btn-visible` class doesn't appear to be used at all.

Reply:
Addressed.